event_email_reminder/models/event.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models
+from datetime import datetime, timedelta
+
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT
+
+
+class EventEvent(models.Model):
+    _inherit = 'event.event'
+
+    @api.model
+    def run_event_email_reminder(
+            self, days=7, near_events=False, template_id=None):
+        """
```

Comment:
Usually [1st line of docstring should be the short description of the method](https://www.python.org/dev/peps/pep-0008/#documentation-strings).